frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -127,6 +127,61 @@
                                             <td>{{ key.submission__submitted_at | number: 0}}&nbsp;{{key.timeSpan}} ago
                                             </td>
                                         </tr>
+
+                                        <tr ng-repeat-end class="collapsible-content_{{challenge.initial_ranking[key.id]}} pointer collapsible-content" style="position: absolute; left: -999em; height: 0px">
+                                            <td style="padding-top: 0px;" colspan="{{key.result.length + 3}}">
```

Comment:
```suggestion
                                            <td style="padding: 0px;" colspan="{{key.result.length + 3}}">
```

Reply:
Lets also keep padding-bottom 0, tested with padding on all sides 0 - This looks good actually.

Reply:
done